bounded-collections/Cargo.toml

```diff
@@ -1,22 +1,23 @@
 [package]
 name = "bounded-collections"
 version = "0.2.3"
-authors = ["Parity Technologies <admin@parity.io>"]
-license = "MIT OR Apache-2.0"
-homepage = "https://github.com/paritytech/parity-common"
 description = "Bounded types and their supporting traits"
-edition = "2021"
-rust-version = "1.79.0"
```

Comment:
maybe worth keeping rust-version though

Reply:
According to https://doc.rust-lang.org/cargo/reference/rust-version.html

> https://doc.rust-lang.org/cargo/reference/rust-version.html

Should we set a workspace level `rust-version = "1.79.0"`, or add `rust-version` back to each crate?

Reply:
I think this is not necessary because downstream seems not obey this field

Reply:
I think each crate is supposed to provide their own MSRV. The main purpose is to give better compiler errors in case someone is using an old toolchain.

Reply:
@niklasad1 thoughts?

Reply:
Sure, a specific rust version per crate sounds good to me since this contain lots of different crates.